plugins/system/tour/tour.php

```diff
@@ -7,10 +7,11 @@
  * @category  Component
  * @package   Joomla.Administrator
  * @author    Joomla! <admin@joomla.org>
- * @copyright (C) 2013 Open Source Matters, Inc. <https://www.joomla.org>
+ * @copyright (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
  * @license   GNU General Public License version 2 or later; see LICENSE.txt
```

Comment:
This is a plugin not a component. Please look at other plugins for the correct examples to follow